great_expectations_cloud/agent/agent.py

```diff
@@ -230,17 +228,29 @@ def _handle_event_as_thread_exit(
         error = future.exception()
         if error is None:
             result: ActionResult = future.result()
-            status = JobCompleted(
-                success=True,
-                created_resources=result.created_resources,
-            )
-            print(f"Completed job: {event_context.event.type} ({event_context.correlation_id})")
+
+            if result.type == UnknownEvent().type:
+                status = JobCompleted(
+                    success=False,
+                    created_resources=[],
+                    error_stack_trace="Unknown event. Ensure agent is up-to-date.",
```

Comment:
Anna left a comment on ZEL-501 suggesting that the error message should read `The version of the GX Agent you are using does not support this functionality. Please upgrade to {version number} or later.` I think we should run the wording past her--might need to tweak her suggested message if we can't get the version number they should update to in there.

Reply:
Set it to closer msg in a0715a9332d0ef27f2e44066119f23f9a281c1fc, confirming with Anna now

Reply:
approved